demo/api/schema.gql

```diff
@@ -691,6 +691,7 @@ input RedirectScopeInput {
 
 type Query {
   currentUser: CurrentUser!
+  currentUserIsAllowed(input: CurrentUserIsAllowedInput!): Boolean!
```

Comment:
should that be a field resolver nested in currentUser?

Reply:
I don't understand your argument - you can have the same aguments in a FieldResolver.